yt_dlp/extractor/clyp.py

```diff
@@ -34,6 +34,21 @@ class ClypIE(InfoExtractor):
         'params': {
             'skip_download': True,
         },
+    }, {
+        'url': 'https://clyp.it/v42214lc',
+        'md5': '4aca4dfc3236fb6d6ddc4ea08314f33f',
+        'info_dict': {
+            'id': 'v42214lc',
+            'ext': 'wav',
+            'title': 'i dont wanna go (old version)',
+            'description': None,
+            'duration': 113.528,
+            'timestamp': 1607348505,
+            'upload_date': '20201207',
+        },
+        'params': {
+            'format': 'wavStreamUrl',
+        },
```

Comment:
```suggestion
```
Shouldn't be needed

Reply:
without this param, for some reason, mp3 is selected

Reply:
Then extractor isn't working properly. Try adding `'ext': 'wav'` to the returned format

Reply:
with `ext` result is the same, but with `acodec` it works correct